{"name": "Прием заказа с некорректным id", "status": "passed", "description": "Принятие заказа с отправкой неверного id курьера", "start": 1701289413136, "stop": 1701289415639, "uuid": "e43d2154-af17-4ee4-93d0-250f88ead959", "historyId": "2d17e0662b60eebdcfc083a8f29d2789", "testCaseId": "2d17e0662b60eebdcfc083a8f29d2789", "fullName": "tests.test_creat_order.TestAcceptOrder#test_order_accept_wrong_courier_id_fail", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_creat_order"}, {"name": "subSuite", "value": "TestAcceptOrder"}, {"name": "host", "value": "DESKTOP-RUURDMU"}, {"name": "thread", "value": "5300-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_creat_order"}]}